app.rb

```diff
@@ -0,0 +1,138 @@
+require './student'
```

Comment:
- [ ] Good job creating the `app.rb` file. However, there's no `main.rb` file. As per [these](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=In%20your%20main,quit%20the%20app.) requirements, The `main.rb` file should define the entry point and show a list of options for the user and act as an entry point for the app.